tachyon/zk/plonk/circuit/selector_compressor.h

```diff
@@ -200,12 +199,12 @@ class SelectorCompressor {
         }
 
         // Update the combination assignment
-        for (auto& [combination, selector] :
-             base::Zipped(combination_assignment, selector.activations())) {
+        const std::vector<bool>& activations = selector.activations();
```

Comment:
Please let me know why `Zipped` was replaced.

Reply:
As specified in the commit message, `Zipped` iterator copies the value. So even though I tried to modify `combination` here, it doesn't change the value in `combination_assignment`.

Reply:
Thank you for the kind explanation!